<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>accountCurator</class>
 <widget class="QDialog" name="accountCurator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir Next Condensed</family>
     <pointsize>25</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;h1&gt;Account&lt;/h1&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>791</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="homeButton">
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mListButton">
      <property name="text">
       <string>Museum list</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="itemsViewedButton">
      <property name="text">
       <string>Items Viewed</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="myCommentsButton">
      <property name="text">
       <string>My Comments</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="accountButton">
      <property name="text">
       <string>Account</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>401</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yuanti TC</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Account Info</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;b&gt;Name:&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="getName">
      <property name="text">
       <string>getName</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>&lt;b&gt;Email:&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="getEmail">
      <property name="text">
       <string>getEmail</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>&lt;b&gt;Username:&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="getUsername">
      <property name="text">
       <string>getUsername</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>&lt;b&gt;Password:&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="getPassword">
      <property name="text">
       <string>getPassword</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>&lt;b&gt;Account Type:&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="getAccountType">
      <property name="text">
       <string>getAccountType</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>401</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yuanti TC</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="title">
    <string>My Museums</string>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>List Museums here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addMusButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>281</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Museum</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Item</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="logoMM">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>0</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="logoMM_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>0</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
